brands/amenity/fast_food.json

```diff
@@ -4531,6 +4531,22 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|庆丰包子铺": {
+    "locationSet": {"include": ["cn"]},
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "庆丰包子铺",
+      "brand:en": "Qingfeng Steamed Dumpling Shop",
+      "brand:wikidata": "Q24841010",
```

Comment:
Is this the correct wikidata entry. It's lacking any fields and will most likely be deleted by the admins there.

Reply:
It is correct. I added an English name to it